content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -60,7 +60,7 @@ different Kubernetes components.
 | `AppArmor` | `true` | Beta | 1.4 | |
 | `BalanceAttachedNodeVolumes` | `false` | Alpha | 1.11 | |
 | `BoundServiceAccountTokenVolume` | `false` | Alpha | 1.13 | 1.20 |
-| `BoundServiceAccountTokenVolume` | `true` | Beta | 1.21 | |
+| `BoundServiceAccountTokenVolume` | `true` | Beta | 1.21 | 1.22 |
```

Comment:
When the feature has graduated or deprecated, please MOVE it rather than COPY it to the second table.

Reply:
done